lib/modules/platform/gitlab/readme.md

```diff
@@ -2,39 +2,44 @@
 
 ## Authentication
 
-First, [create a Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) for the bot account.
+You can authenticate Renovate to GitLab, with a Personal Access Token, or Group Access Token.
+
+To start, create either:
+
+- a [Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) for the bot account
+- or a [Group Access Token](https://docs.gitlab.com/ee/user/group/settings/group_access_tokens.html#bot-users-for-groups) for the bot account
 
 If you are using a Group access token, the token must have Developer role or higher permissions in order to create issues and merge requests.
 The token must have Maintainer permissions in order to perform Automerge.
+To keep using the same GitLab-generated bot user you must [rotate/refresh the Group Access Token](https://docs.gitlab.com/ee/api/group_access_tokens.html#rotate-a-group-access-token) _before_ the token's expiry date.
```

Comment:
Is this whole part about group access tokens? Right now we have this structure:

- One line about group access tokens needing "developer role or higher permissions"
- One line about tokens needing Maintainer permission to automerge
- One line about refreshing/rotating tokens

It's not clear to me when we're talking about Group access tokens, and when about generic instructions that apply to _both_ Group access tokens and personal access tokens.

Can we split up the text into clearer paragraphs?

Reply:
You are right. It is actually mixed up right now. The role refers to the role of the bot account (so it affects both cases).

How about the following:

> The bot account must have at least the Developer role in order to create issues and merge requests.
> The bot account must have the Maintainer role in order to perform automerge.
>
> If you are using a group access token, to keep using the same GitLab-generated bot user you must [rotate/refresh the Group Access Token](https://docs.gitlab.com/ee/api/group_access_tokens.html#rotate-a-group-access-token) _before_ the token's expiry date.

Reply:
The new text looks good to me!